lib/tests/container.rs

```diff
@@ -65,16 +49,75 @@ impl Neo4jContainer {
             .0
     }
 
-    fn version_from_env() -> String {
+    fn server_from_env() -> (TestServer, String) {
+        const USER_VAR: &str = "NEO4J_TEST_USER";
+        const PASS_VAR: &str = "NEO4J_TEST_PASS";
+        const TEST_URI_VAR: &str = "NEO4J_TEST_URI";
         const VERSION_VAR: &str = "NEO4J_VERSION_TAG";
+
+        const DEFAULT_USER: &str = "neo4j";
+        const DEFAULT_PASS: &str = "neo";
```

Comment:
could that fail in 5.x that introduced the min-password length of 8 ?

Reply:
It would, but we do this to allow it

https://github.com/neo4j-labs/neo4rs/blob/4dad182f26dbb501ab9a58c38ef3b677cebe4e96/lib/tests/container.rs#L89-L92